File: Server-Side/FlightBoard/FlightBoard.Domain/Entities/Flight.cs
using FlightBoard.Domain.ValueObjects;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FlightBoard.Domain.Entities
{
    public class Flight
    {
        public Guid Id { get; set; }
        public string FlightNumber { get; set; } = null!;
        public string Destination { get; set; } = null!;
        public DateTime DepartureTime { get; set; }
        public string Gate { get; set; } = null!;
    }
}

File: Client-Side/flightboard-client/src/types/Flight.ts
import { FlightStatus } from "./FlightStatus";

export type FlightDto = {
    flightNumber: string;
    destination: string;
    departureTime: string; 
    gate: string;
  }

  export type Flight = FlightDto & {
    id: string;
    status: FlightStatus;
  };